cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -526,20 +526,25 @@ define([
                 expect(this.view.$el).toContainText(emptyMessage);
                 expect(this.view.$(SELECTORS.itemView)).not.toExist();
             });
+
+            it('can create a new group configuration', function () {
+                this.view.$('.new-button').click();
+                expect($('.group-configuration-edit').length).toBeGreaterThan(0);
+            });
         });
     });
 
-    describe('GroupConfigurationItem', function() {
+    describe('GroupConfigurationItemView, the controller view for experiment group configurations', function() {
```

Comment:
I'm not loving these new long names for test suites. I think they will make it harder to read when running the tests in the browser. I did a quick look around other tests and we are inconsistent in how we name suites. Here are some examples:

    describe("ContainerPage", function() {
        describe("Initial display", function() {
        describe("Duplicating an xblock", function() {

    describe("Cohorts View", function () {
        describe("Add Cohorts Form", function () {
        describe("Add Students Button", function () {

I like the way this tutorial names suites:

http://tinnedfruit.com/2011/03/03/testing-backbone-apps-with-jasmine-sinon.html